packages/account/src/Helpers/utils.tsx

```diff
@@ -77,7 +77,10 @@ export const getDocumentData = (country_code: string, document_type: string) =>
         example_format: '',
     };
     const IDV_DOCUMENT_DATA: any = getIDVDocuments(country_code);
-    return IDV_DOCUMENT_DATA[document_type] ?? DEFAULT_CONFIG;
+    if (IDV_DOCUMENT_DATA) {
```

Comment:
how `IDV_DOCUMENT_DATA` can be null tho? 
It's like we're not getting the expected `country_code` but the question is why are we getting different country_code than expected!

Reply:
Please check the function `getIDVDocuments`, u'll see that this function is used to return custom config that is required for IDV document fields. Some configs are handled by FE STILL. When we do not get the custom config for the country, the `IDV_DOCUMENT_DATA` is undefined. A null check is added to return default config which will later by updated with required values from BE API response